platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/pax/web/jetty/DelegateServletFilter.java

```diff
@@ -1,54 +0,0 @@
-/**
- * Copyright (c) Codice Foundation
- *
- * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
- * Lesser General Public License as published by the Free Software Foundation, either version 3 of
- * the License, or any later version.
- *
- * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
- * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
- * License is distributed along with this program and can be found at
- * <http://www.gnu.org/licenses/lgpl.html>.
- */
-package org.codice.ddf.pax.web.jetty;
-
-import java.io.IOException;
-import java.util.List;
-import javax.servlet.Filter;
-import javax.servlet.FilterChain;
-import javax.servlet.FilterConfig;
-import javax.servlet.ServletException;
-import javax.servlet.ServletRequest;
-import javax.servlet.ServletResponse;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class DelegateServletFilter implements Filter {
```

Comment:
❓ Isn't this going to break metrics and other cross-cutting concerns? I think we have some dependents on this class without any viable substitutes as of yet.

Reply:
> Isn't this deletion going to break metrics and other cross-cutting concerns (i.e. client info filter but I doubt anyone still cares about that)? I think we have some dependents on this class without any viable substitutes as of yet.

I only changed the name of this class. It's now `DelegateSecurityFilter`, but it still performs the same behavior, picking up all registered ServletFilters and passing incoming requests off to them.

And what do you mean by "dependencies"? I wasn't aware anybody used this class itself. They should just be registering ServletFilters which will get picked up by this.  

> I don't think your SecurityFilter workaround is valid in all current cases.

Do you have an example by chance where this wouldn't work?

Reply:
You were right. The DoPrivilegedFilter wasn't working either, since the doPrivileged call terminated as soon as all the Filters had run. The delegate filter wasn't really running the the filters as part of the chain.

I believe I fixed that now, so marking this as resolved